internal/cli/atlas/backup/schedule/update.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schedule
+
+import (
+	"context"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+var updateTemplate = "Snapshot backup policy for cluster '{{.ClusterName}}' updated.\n"
+
+type UpdateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	clusterName string
+	store       store.ScheduleUpdater
+}
+
+func (opts *UpdateOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *UpdateOpts) Run() error {
+	r, err := opts.store.UpdateSchedule(opts.ConfigProjectID(), opts.clusterName, &atlas.CloudProviderSnapshotBackupPolicy{})
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func UpdateBuilder() *cobra.Command {
+	opts := &UpdateOpts{}
+	cmd := &cobra.Command{
+		Use:     "update",
```

Comment:
since this command is incomplete make it hidden until the latest PR is ready and merged
```suggestion
		Use:     "update",
		Hidden: true,

```

Reply:
Hidden! 🥷